gr4vy.go

```diff
@@ -22,6 +22,7 @@ type Gr4vyClient struct {
 	Debug       bool
 	accessToken string
 	environment string
+	merchantId string
```

Comment:
We've used `merchantAccountId` every where to make it clear this is not their PSP / Acquirer MID.

```suggestion
	merchantAccountId string
```